styles.css

```diff
@@ -1,33 +1,58 @@
 :root {
     --menu-bar-width: 60px;
     --menu-bar-height: 10px;
+    --main-color: hsl(334, 66%, 55%);
 }
 
+/* div blokjes zichtbaar maken */
 #menu {
-    background-color: #D84282;
-    position: relative;
+    /* met de eerste 3 zetten we een balk, want zonder content in index krijg je niks te zien*/
+    background-color: var(--main-color);
     width: var(--menu-bar-width);
     height: var(--menu-bar-height);
-    margin: 100px;
+    margin: 50px;
+    position: relative;
 }
 
+/*pseudo-elementen before en after*/
 #menu::before,
 #menu::after {
-    background-color: #D84282;
-    display: block;
-    content: "";
+    /*met de eerste 3 zetten  we een balk, zonder content in index krijg je niks te zien*/
+    background-color: var(--main-color);
     width: var(--menu-bar-width);
     height: var(--menu-bar-height);
+    /*pseudo-element moet altijd een content hebben om zichtbaar te zijn*/
+    content: "";
+    /*zet deze er altijd bij*/
+    display: block;
+    /*  zonder absolute/relative zitten de blokjes over elkaar heen*/
     position: absolute;
 }
 
+/*hiermee zet je de balken onder en boven op de juiste plek*/
 #menu::before {
     left: 0;
     top: -20px;
 }
 
 #menu::after {
     left: 0;
-    top: 20px;
+    bottom: -20px;
+}
+
+/*code voor het kruisje*/
+#menu:hover {
```

Comment:
Je hebt de hover op je menu gezet, maar dit alleen het middelste segment en daarom wordt het menu geen kruisje als ik uitsluitend over de bovenste of onderste balk hover.